sparse/tests/test_core.py

```diff
@@ -600,3 +600,31 @@ def test_scalar_slicing():
     s = COO(x)
     assert_eq(x[0], s[0])
     assert_eq(x[1], s[1])
+
+
+@pytest.mark.parametrize('shape, k', [
+    ((3, 4), 0),
+    ((3, 4, 5), 1),
+    ((4, 2), -1),
+    ((2, 4), -2),
+    ((4, 4), 1000),
+])
+def test_triu(shape, k):
+    x = random_x(shape)
+    s = COO.from_numpy(x)
+
+    assert_eq(np.triu(x, k), s.triu(k))
+
+
+@pytest.mark.parametrize('shape, k', [
+    ((3, 4), 0),
+    ((3, 4, 5), 1),
+    ((4, 2), -1),
+    ((2, 4), -2),
+    ((4, 4), 1000),
+])
+def test_tril(shape, k):
+    x = random_x(shape)
+    s = COO.from_numpy(x)
+
+    assert_eq(np.tril(x, k), s.tril(k))
```

Comment:
We might include each of these assert lines into a single `test_trilu` test?

Reply:
I guess I could do that, yes.